sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -83,6 +83,8 @@ export class SupportingUserAESTController {
       firstName: supportingUserForApplication.user?.firstName,
       lastName: supportingUserForApplication.user?.lastName,
       hasValidSIN: supportingUserForApplication.personalInfo?.hasValidSIN,
+      parentFullName: supportingUserForApplication.fullName,
+      personalInfo: supportingUserForApplication.personalInfo,
```

Comment:
The `personalInfo` should not be returned directly from the API to avoid the future additions to the JSONB will not flow directly to the client.

Please do the following:
- Create a DTO for the SupportingUserPersonalInfo to be used under `SupportingUserFormDataAPIOutDTO {
  formName: string;`.
- Map each property in line 87, as shown below.
```ts
personalInfo: {
  givenNames: supportingUserForApplication.personalInfo?.givenNames,
  lastName: supportingUserForApplication.personalInfo?.lastName,
  hasValidSIN: supportingUserForApplication.personalInfo?.hasValidSIN,
},
```